akka-stream/src/main/mima-filters/2.6.15.backwards.excludes/30246-boundary-event-cancellation-hook.excludes

```diff
@@ -0,0 +1,2 @@
+# Changes to internal/private
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.impl.fusing.ActorGraphInterpreter#BoundaryEvent.cancel")
```

Comment:
Should be in 2.6.14 backwards excludes, latest released version. Curious that the CI isn't failing.

Reply:
To be honest, I never know which versions to pick. I always have to look around. 

I think the reason is that `2.6.15.backwards.excludes` reads as: what are the exclusions that we version 2.6.15 had to to add. 

No idea why CI didn't complain.